src/pages/gateway/headers.md

```diff
@@ -86,3 +86,30 @@ This can be useful for authorization, authentication, and tracking headers that
    -  **Value**: `new-trackingId`
 -  **Key**: `GGW-SH-differentSource-trackingId`
    -  **Value**: `different-trackingId`
+
+## Add response headers
+
+Mesh owners can use `responseConfig` to add response headers to their mesh. `responseConfig` must contain a `headers` field that contains your headers as key value pairs.
+
+``` json
+    { 
+      "meshConfig": { 
+        "sources": [
+          {
+            "name": "venia", 
+            "handler": { 
+              "graphql": { 
+                "endpoint": "https://venia.magento.com/graphql"
+              } 
+            } 
+          }
+        ],
+       "responseConfig": {
+       "headers": {
```

Comment:
I think the indentation got thrown off for this section (unless Github's just being weird for me). Can you please double check this?

Reply:
Thanks, @amolina-magento! I believe I have corrected the issue.